apps/researcher/src/lib/community.ts

```diff
@@ -104,3 +158,17 @@ export async function joinCommunity({
     role: 'basic_member',
   });
 }
+
+export async function editDescription({
```

Comment:
The term 'edit', to me, doesn't necessarily mean that data is going to be stored. Change the term to e.g. 'save' or 'update'?